pkg/cache/cache.go

```diff
@@ -42,9 +42,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client/apiutil"
 )
 
-var (
-	defaultSyncPeriod = 10 * time.Hour
-)
+var defaultSyncPeriod = 10 * time.Hour
```

Comment:
Nit: Please avoid stylistic no-op changes, they cluster the git history for no good reason IMHO